google-cloud-storage/samples/storage_disable_bucket_lifecycle_management.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LCC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def disable_bucket_lifecycle_management bucket_name:
+  # [START storage_disable_bucket_lifecycle_management]
+  # Disable lifecycle management for a bucket
+  # bucket_name = "Your Google Cloud Storage bucket name"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new
+  bucket = storage.bucket bucket_name
+
+  rules = bucket.lifecycle(&:clear)
+
+  puts "Lifecycle management is disabled for bucket #{bucket_name} and the rules are #{rules}"
```

Comment:
Won't rules always be null here? Might look confusing to print it out, maybe we can just drop it

Reply:
Removed the second part of the sentence and the `rules` variable.